src/ggsmark.js

```diff
@@ -51,7 +51,43 @@ export default function (text) {
       node.unlink()
     }
     // if (node.type === 'text' && !!node.parent && node.parent.type === 'paragraph')
+    else if(
+    node.type === 'text' &&
+    !!node.parent &&
+    node.parent.type === 'paragraph' &&
+    node.literal.match(/\:text-center/)
+  ) {
+    let nestedEvent = walker.next()
+    let nestedNode = nestedEvent.node
+
+    // Make sure text nodes are not fragmented
+    while (!!nestedNode.prev && nestedNode.type === 'text') {
+      nestedNode.prev.literal = nestedNode.prev.literal + nestedNode.literal
+      nestedNode.unlink()
+      nestedEvent = walker.next()
+      nestedNode = nestedEvent.node
+    }
+
+    let matchTextCntrExp = (/\:text-center/)
+    let splitText = node.literal.split(matchTextCntrExp)
+
+    for (let index in splitText) {
+      if (index % 2 == 0) {
+        let text = new Node('text')
+        text.literal = splitText[index]
+        node.insertBefore(text)
+      } else {
+        let div = new Node('custom_block')
+        div.onEnter = `<div class="text-center" style="text-allign: center"> ${splitText[index]}`
```

Comment:
```suggestion
        div.onEnter = `<div style="text-allign: center"> ${splitText[index]}`
```
Perhaps we can remove the class since the style will overwrite it anyways